lib/DidJwtStrategy.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+const passport = require('passport');
+const util = require('util');
+const validate = require('bedrock-validation').validate;
+const BedrockError = bedrock.util.BedrockError;
+
+module.exports = Strategy;
+
+/**
+ * Creates a new didJwt Strategy for use with passport.
+ */
+function Strategy(options) {
+  const v = validate('authn-did-jwt.constructor', options);
+  if(!v.valid) {
+    throw v.error;
+  }
+  passport.Strategy.call(this);
+  this.name = 'didJwt',
+  this.options = options;
+  this.store = {};
+}
+util.inherits(Strategy, passport.Strategy);
+
+/**
+ * Authenticate a request.
+ *
+ * @param req the request to authenticate.
+ */
+Strategy.prototype.authenticate = function(req) {
+  const self = this;
+  self.store.verify({
+    namespace: self.options.namespace,
+    token: req.body.token
```

Comment:
There is a standard for JWT authentication that uses the `Authorization: Bearer <token>` HTTP header. We should follow this pattern unless we've got a good reason not to.